src/style.css

```
@@ -0,0 +1,40 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
```

you can just remove margin-right: 4px; from .stars__star and make like this
```suggestion
.stars__star:not(:last-child) {
  margin-right: 4px;
}
```